vscode-extensions.json

```diff
@@ -191,7 +191,11 @@
       "revision": "0.2.0"
     },
     {
-      "repository": "https://github.com/Microsoft/vscode-node-debug2",
+      "repository": "https://github.com/microsoft/vscode-eslint",
+      "revision": "release/2.1.0-next.1"
```

Comment:
Let's use a commit ID here if there is no corresponding release tag. I just looked for the commit where the extension's `package.json` version was bumped to 2.1.1: https://github.com/microsoft/vscode-eslint/commit/1e15d3495da89072d48cf583d48d92829f0c4b82

The extension I built [here](https://download.jboss.org/jbosstools/vscode/3rdparty/vscode-eslint/vscode-eslint-2.1.1-1e15d3.vsix) is based on that commit.

Reply:
Thanks a lot. I just thought that this moment is not pretty important and used the closest commit.

Reply:
Done